app/controllers/local_petitions_controller.rb

```diff
@@ -0,0 +1,30 @@
+require 'postcode_sanitizer'
+
+class LocalPetitionsController < ApplicationController
+  respond_to :html
+
+  def index
```

Comment:
What would you think of an approach for another action doing the postcode check and redirecting to a `show` action here.

We could then even leverage friendly id's and have something like `petitions/local/westminster-north` which would be a meaningful URL someone could share.